### ISSUE TITLE
Multiple `set-cookie` headers from endpoints returned as single value

### ISSUE THREAD

**Issue by coderpradp** [yarbsemaj/sveltekit-adapter-lambda#15]
Currently `set-cookie` header with multiple value (array) is being returned as comma separated single value.
Browsers can't interpret it correctly and only sets first value before comma in browser cookie.

adapter-netlify ([code](https://github.com/sveltejs/kit/blob/c660143ad2b90ea3a1ae18247c553da2538750a0/packages/adapter-netlify/src/headers.js#L20)) seems to solve this by returning multivalued set-cookie header without joining it into single value.

Will it be possible for this adapter to support this in similar way?

Edit: Added Reproduction
This endpoint:

```
export function get() {
  return {
    headers: {
      'set-cookie': [
        'accesstoken=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT; HttpOnly;',
        'refreshtoken=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT; HttpOnly;'
      ]
    }
  }
}
```
produces the following headers:

```
set-cookie: accesstoken=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT; HttpOnly, refreshtoken=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT; HttpOnly
```
but it should produce:

```
set-cookie: accesstoken=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT; HttpOnly
set-cookie: refreshtoken=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT; HttpOnly
```

**Comment by yarbsemaj** [comment]
Hi @coderpradp Thank You for your interest in my project.
I have just published a new version of the adapter 0.9.3 that  I believe fixes this issue, please let me know if not 😄

**Comment by coderpradp** [comment]
@yarbsemaj Thank you. It is working perfectly now.
